packages/roosterjs-editor-types/lib/enum/ClipboardSource.ts

```diff
@@ -0,0 +1,10 @@
+export const enum ClipboardSource {
```

Comment:
add comment to this enum and all members. Public members need comment

Reply:
Since these values are only used in one single file now, Do we still need to make them public?